x-pack/plugins/cases/public/components/user_actions/index.test.tsx

```diff
@@ -404,23 +404,24 @@ describe(`UserActions`, () => {
     });
 
     it('shows the correct username', async () => {
-      const isolateAction = [getHostIsolationUserAction()];
+      const isolateAction = [
+        getHostIsolationUserAction({ createdBy: { profileUid: userProfiles[0].uid } }),
+      ];
       const props = {
         ...defaultProps,
+        userProfiles: userProfilesMap,
         caseUserActions: isolateAction,
-        data: { ...defaultProps.data, comments: [hostIsolationComment()] },
+        data: {
+          ...defaultProps.data,
+          comments: [hostIsolationComment({ createdBy: { profileUid: userProfiles[0].uid } })],
+        },
       };
 
-      const wrapper = mount(
-        <TestProviders>
-          <UserActions {...props} />
-        </TestProviders>
-      );
-      await waitFor(() => {
-        expect(wrapper.find(`[data-test-subj="user-action-avatar"]`).first().prop('name')).toEqual(
-          defaultProps.data.createdBy.fullName
-        );
-      });
+      appMockRender.render(<UserActions {...props} />);
+      // there are two because one is for the
```

Comment:
nit: for the? 🙂

Reply:
No idea what I was going for here haha, will remove.